### DISCUSSION TITLE
org-journal

### DISCUSSION THREAD

**Discussion by danilshvalov** [nvim-orgmode/orgmode#551] [Ideas]
I'm working on an [`org-journal`](https://github.com/bastibe/org-journal) port for neovim. Do you see it as part of the core or as a separate plugin? If you are interested, I can create a PR.

**Comment by kristijanhusak** [comment]
Generally, we try to follow the same route as Emacs. If something is part of the core in Emacs, we can make it part of the core here. Since it seems that Emacs has the separate plugin you linked, we should also have it as a separate plugin here.
